app.py

```diff
@@ -367,10 +367,15 @@ def render_chatbox_header(self):
                 is_dm_workaround_please_remove_me=True
             )
         else:
+            are_more_members = False
+            if self.store.state.members['response_metadata']:
```

Comment:
Are you sure that `response_metadata` is always present? Otherwise, it would break, and a `.get` with a default value would be necessary.